daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -82,13 +85,74 @@ Logs for application and `daprd` are captured in separate files. These log files
 
 Even if you've decided to rename your resources folder to something other than `.dapr`, the log files are written only to the `.dapr/logs` folder (created in the application directory).
 
-
 ## Watch the demo
 
 Watch [this video for an overview on Multi-App Run](https://youtu.be/s1p9MNl4VGo?t=2456):
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/s1p9MNl4VGo?start=2456" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+{{% /codetab %}}
+
+{{% codetab %}}
+<!--kubernetes-->
+
+## Multi-App Run template file
+
+When you execute `dapr run -f -k .`, Dapr generates the multi-app template file (named `dapr.yaml`) in the `.dapr/deploy` directory within each application. Each time you run `dapr run -f -k .`, the generated Kubernetes Deployment YAML file is overwritten. 
```

Comment:
When the `dapr.yaml` template file is given as input (directory via `dapr run -k -f .`) or `dapr run -k -f dapr.yaml`, the necessary default service and deployment definitions for K8s are generated within the `.dapr/deploy` folder for each app in the `dapr.yaml` template. 
If `createService` field is present in the `dapr.yaml` template then `service.yaml` and the `deployment.yaml` files will be generated in the `.dapr/deploy` folder of each app in the multi-app run template and will be used to deploy the applications in a dev/test environment in K8s. 

For now the apps and services will be deployed in the `default` namespace of K8s.

Reply:
@mukundansundar Updated - lmk what you think?